src/content/app-architecture/design-patterns/sql.md

```diff
@@ -58,7 +58,7 @@ and restored when the user starts the application.
 
 :::note
 The full, runnable source-code for this example is
-available in [`/examples/cookbook/architecture/todo_data_service/`][].
+available in [`/examples/app-architecture/todo_data_service/`][].
```

Comment:
Thanks for noticing this and opening this PR to update the link!

However, you'll need to update the corresponding Markdown link definition on line 486 as well. Let me know if you have any issues doing so :)